### DISCUSSION TITLE
Running the "pac" command in Windows 11 through Parallels on a MacBook Pro

### DISCUSSION THREAD

**Discussion by GorgonUK** [microsoft/powerplatform-build-tools#888] [Q&A]
Whenever I would run the PAC command in PowerShell I would get this error:

[ERROR] Unhandled Exception: System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
[ERROR]    at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
[ERROR]    at System.Reflection.RuntimeModule.GetTypes()
[ERROR]    at System.Reflection.Assembly.GetTypes()

Running the "pac install latest" did no good.

I managed to resolve the error this way:
1. Navigate to https://nuget.org 
2. Download the "Microsoft.PowerApps.CLI" package manually - Now you should have microsoft.powerapps.cli.x.xx.x.nupkg in your "Downloads" folder
3. Copy the nupkg file to the existing folder C:\Users\user\AppData\Local\Microsoft\PowerAppsCLI/Microsoft.PowerApps.CLI.x.xx.x
4. Run "pac install latest" again in PowerShell

This might be useful for people who want to develop PCF controls on MacOS

**Comment by petrochuk** [comment]
Here is how you can [install PAC CLI](https://learn.microsoft.com/en-us/power-platform/developer/cli/introduction?tabs=windows#install-microsoft-power-platform-cli). For MacOS and Linux it might be easier to use `dotnet tool` [to install PAC CLI](https://learn.microsoft.com/en-us/power-platform/developer/howto/install-cli-net-tool?tabs=macos)
